electron-app/src/views/settings/UserSecuritySettings.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <v-container id="settings-user-security">
+    <v-row no-gutters>
+      <v-col>
+        <v-card>
+          <v-card-title>User Account & Security</v-card-title>
+          <v-form ref="form">
+            <v-card-text>
+              <revealable-input
+                v-model="currentPassword"
+                class="settings-general__account-and-security__fields__current-password"
```

Comment:
Shouldn't this be `user-security-settings__fields__current-password`?

Reply:
Yes good call, done.